testsupport/tiers/checks.go

```diff
@@ -1282,7 +1282,7 @@ func appstudioUserActionsRole() spaceRoleObjectsCheck {
 				},
 				{
 					APIGroups: []string{"managed-gitops.redhat.com"},
-					Resources: []string{"gitopsdeployments"},
+					Resources: []string{"gitopsdeployments", "gitopsdeploymentmanagedenvironments", "gitopsdeploymentrepositorycredentials", "gitopsdeploymentsyncruns"},
 					Verbs:     []string{"get", "list", "watch"},
```

Comment:
you changed the template so the admin has all permissions
```suggestion
					Verbs:     []string{"*"},
```

Reply:
Good catch @MatousJobanek